Makefile

```diff
@@ -118,43 +118,13 @@ $(BIN_DIR)/azure-acr-credential-provider.exe: $(PKG_CONFIG) $(wildcard cmd/acr-c
 ##@ Images
 ## --------------------------------------
 
-.PHONY: docker-pull-prerequisites
-docker-pull-prerequisites: ## Pull prerequisite images.
-	docker pull docker/dockerfile:1.3.1
-	docker pull docker.io/library/golang:1.17-buster
-	docker pull gcr.io/distroless/static:latest
-
 buildx-setup:
 	docker buildx inspect img-builder > /dev/null || docker buildx create --name img-builder --use
 	# enable qemu for arm64 build
 	# https://github.com/docker/buildx/issues/464#issuecomment-741507760
 	docker run --privileged --rm tonistiigi/binfmt --uninstall qemu-aarch64
 	docker run --rm --privileged tonistiigi/binfmt --install all
 
-.PHONY: build-ccm-image
-build-ccm-image: buildx-setup docker-pull-prerequisites ## Build controller-manager image.
```

Comment:
Could we keep this or refine it to build for the specified arch/platform? The internal pipeline may depend on this (build and export to tar).

Reply:
will add another group to build ccm

Reply:
build-ccm-image-% target is added. e.g `make build-ccm-image-amd64`

Reply:
same case for cnm, and we already use the following syntax to build cnm to mcr.
`OUTPUT_TYPE="docker,dest=$mydir/azure-cloud-node-manager.tar" ARCH=$(arch) make build-node-image-linux`